mautic/mautic
# fix: attribution tooltip meaning

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This update addresses a critical issue with the Attribution field tooltips in Mautic's contact management system. The previous descriptions were misleading, incorrectly suggesting that Attribution referred to marketing channels rather than revenue.

The revised tooltips now reflect the purpose of these fields. "Attribution" is clarified as the estimated revenue generated by a lead, while "Attribution Date" is defined as the date when this revenue was realized or should be credited.

Check before and after in the Changed files tab.






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts
3. Create new
4. See the field and hover with the cursor to see the toolti[
